tsl/src/chunk.c

```diff
@@ -192,6 +188,7 @@ chunk_set_default_data_node(PG_FUNCTION_ARGS)
 int
 chunk_invoke_drop_chunks(Oid relid, Datum older_than, Datum older_than_type)
 {
+	char *const function_name = "drop_chunks";
```

Comment:
Could you please retain DROP_CHUNKS_FUNCNAME macro?

Reply:
This is  more a stylistic than correctness. Our code base uses macros for string constants (a standard practice for numeric and string constants in C). The original code actually has it and here you are undoing that.